axis.go

```diff
@@ -39,3 +39,47 @@ func (keys AxisKeyPair) Value() float32 {
 
 	return AxisNeutral
 }
+
+const (
+	// The vertical mouse axis
+	AxisMouseVert = 0
+	// The horizontal mouse axis
+	AxisMouseHori = 1
+)
+
+// AxisMouse is an axis of the mouse direction.
+type AxisMouse struct {
+	direction int
+
+	old float32
```

Comment:
Internal documentation for those working or browsing through engo, not clear at first what this does.